### PR TITLE
[test] Add more tests for unsigned LEB128 with non-minimal length

### PR DESCRIPTION
**Pull request by Honry**
tweak some tests order a bit to make it looks beauty passingly.

### CODE REVIEW

File: test/core/binary.wast [hunk 1]
```diff
@@ -6,6 +6,9 @@
 (assert_malformed (module binary "") "unexpected end")
 (assert_malformed (module binary "\01") "unexpected end")
 (assert_malformed (module binary "\00as") "unexpected end")
+(assert_malformed (module binary "\00asm") "unexpected end")
+(assert_malformed (module binary "\00asm\01") "unexpected end")
+(assert_malformed (module binary "\00asm\01\00\00") "unexpected end")
```
Comment (rossberg) [inline test/core/binary.wast]: I'd prefer to keep this where they were before. That was grouping all tests concerned with the version.

Comment (Honry) [reply test/core/binary.wast]: Sure, done. :)

File: test/core/binary.wast [hunk 2]
```diff
@@ -37,6 +34,9 @@
 (assert_malformed (module binary "\ef\bb\bf\00asm\01\00\00\00") "magic header not detected")
 
 ;; Unknown binary version.
```
Comment (rossberg) [inline test/core/binary.wast]: Nit: can change this heading to "malformed binary version"?

Comment (Honry) [reply test/core/binary.wast]: Yeah, it‘s more explicit, thanks!

**Pull request merged**